test/refund.js

```diff
@@ -0,0 +1,99 @@
+const { assertRevert } = require('@aragon/court/test/helpers/asserts/assertThrow')
```

Comment:
Given that this directly modifies behaviour (and is completely untested) in the extended contracts, could we make this test runnable from other tests (e.g. see [Vault's](https://github.com/aragon/aragon-apps/blob/master/apps/vault/test/vault_shared.js); you just need to move the `artifact` calls to be inside the exported function)?

Reply:
Yes, I actually considered doing something like this. Makes sense.